### PR TITLE
Update RetinaNet + RCNN to use 'classification' instead of cls

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

File: keras_cv/models/object_detection/retina_net/retina_net.py [hunk 1]
```diff
@@ -350,15 +350,15 @@ def compute_loss(self, images, boxes, classes, training):
         box_weights = positive_mask / normalizer
         y_true = {
             "box": boxes,
-            "cls": cls_labels,
+            "classification": cls_labels,
```
Comment (ianstenbit) [inline keras_cv/models/object_detection/retina_net/retina_net.py:353]: should we also s/cls/class for variable names if we're making this change?

Comment (LukeWood) [reply keras_cv/models/object_detection/retina_net/retina_net.py:353]: lets do it, but lets keep it self contained for now as this change is more important.  This change shows up in user facing APIs - variable names don't.

Comment (ianstenbit) [reply keras_cv/models/object_detection/retina_net/retina_net.py:353]: Where does this show up in APIs? I assume it shows up in logs as a loss name?

**Review by ianstenbit — APPROVED** [review]

**Pull request merged**